fenix/app/src/androidTest/java/org/mozilla/fenix/ui/robots/CustomTabRobot.kt

```diff
@@ -106,10 +132,23 @@ class CustomTabRobot {
 
     fun waitForPageToLoad() = progressBar.waitUntilGone(waitingTime)
 
+    fun clickCustomTabCloseButton() = closeButton().click()
+
+    fun verifyCustomTabActionButton(customTabActionButtonDescription: String) =
+        assertItemWithDescriptionExists(itemWithDescription(customTabActionButtonDescription))
+
+    fun verifyPDFReaderToolbarItems() =
+        assertItemWithResIdAndTextExists(
+            itemWithResIdAndText("download", "Download"),
+            itemWithResIdAndText("openInApp", "Open in app"),
+        )
```

Comment:
@AndiAJ is it possible to use `"$packageName:id` here, if so we can we swap?

Reply:
Hi @AaronMT , unfortunately no, we'll need to use the hardcoded strings in this case.
There are no res Ids for the PDF toolbar buttons, and also there aren't any strings that could be used as resources.